index.html

```diff
@@ -28,12 +28,30 @@
           <li class="nav-item nav-link"><a href="#">About</a></li>
           <li class="nav-item nav-link"><a href="#">Contact</a></li>
         </ul>
-        <div class="hambuger">
-          <span class="bar"></span>
-          <span class="bar"></span>
-          <span class="bar"></span>
-        </div>
       </nav>
+        <div class="container12"></div>
+        <div class="navbar">
+          <div class="hamburger-menu">
+            <div class="line  line-1"></div>
+            <div class="line line-2"></div>
+            <div class="line line-3"></div>  
+            </div>
```

Comment:
- [ ] Kindly be noticed that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.